<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Anasayfa</class>
 <widget class="QMainWindow" name="Anasayfa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#ddfeff;
font-size:14pt;
color: rgb(170, 0, 0);
font-weight:bold;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="urun">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>751</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ddfeff;
color: rgb(170, 0, 255);
font-size:14pt;
font: 8pt &quot;Comic Sans MS&quot;;
font-weight:bold;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Çanta</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>771</width>
        <height>521</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="c1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>191</width>
         <height>221</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>C:/Users/Dell/.designer/backup/foto/1.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="c2">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>100</y>
         <width>191</width>
         <height>221</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>C:/Users/Dell/.designer/backup/foto/2.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="c3">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>100</y>
         <width>191</width>
         <height>221</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>C:/Users/Dell/.designer/backup/foto/3.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="bs1">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>410</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 255, 127);
font: 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>Sepete 
 ekle</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>340</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;


</string>
       </property>
       <property name="text">
        <string>470,50TL</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>340</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;


</string>
       </property>
       <property name="text">
        <string>690,95TL</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>340</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;


</string>
       </property>
       <property name="text">
        <string>450,95TL</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="bs2">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>410</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 255, 127);
font: 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>Sepete 
 ekle</string>
       </property>
      </widget>
      <widget class="QPushButton" name="bs3">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>420</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 255, 127);
font: 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>Sepete 
 ekle</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Ayakkabı</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>771</width>
        <height>521</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="a1">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>191</width>
         <height>221</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>C:/Users/Dell/.designer/backup/foto/4.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="a2">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>100</y>
         <width>191</width>
         <height>221</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>C:/Users/Dell/.designer/backup/foto/5.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="a3">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>100</y>
         <width>191</width>
         <height>221</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>C:/Users/Dell/.designer/backup/foto/6.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="bs4">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>420</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 255, 127);
font: 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>Sepete 
 ekle</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>340</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;


</string>
       </property>
       <property name="text">
        <string>550,20TL</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>320</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
font: 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
text-decoration: line-through;


</string>
       </property>
       <property name="text">
        <string>1090TL</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>340</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;


</string>
       </property>
       <property name="text">
        <string>720,95TL</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="bs5">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>420</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 255, 127);
font: 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>Sepete 
 ekle</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>350</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;


</string>
       </property>
       <property name="text">
        <string>920TL</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="bs5_2">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>420</y>
         <width>61</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>9</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 255, 127);
font: 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>Sepete 
 ekle</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>180</y>
      <width>251</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 13pt &quot;MS Serif&quot;;
color: rgb(0, 0, 127);</string>
    </property>
    <column>
     <property name="text">
      <string>Ürün</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fiyat</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>759</x>
      <y>245</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 25 10pt &quot;Cascadia Code Light&quot;;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>882</x>
      <y>245</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 25 10pt &quot;Cascadia Code Light&quot;;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>883</x>
      <y>330</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 25 10pt &quot;Cascadia Code Light&quot;;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>330</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 25 10pt &quot;Cascadia Code Light&quot;;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1006</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMen">
    <property name="title">
     <string>Menü</string>
    </property>
    <addaction name="separator"/>
    <addaction name="cikis"/>
   </widget>
   <widget class="QMenu" name="menuH_sab_m">
    <property name="title">
     <string>Hısabım</string>
    </property>
    <addaction name="kayit"/>
   </widget>
   <addaction name="menuMen"/>
   <addaction name="menuH_sab_m"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionKad_n">
   <property name="text">
    <string>Kadın</string>
   </property>
  </action>
  <action name="actionErkek">
   <property name="text">
    <string>Erkek</string>
   </property>
  </action>
  <action name="action_ocuk">
   <property name="text">
    <string>Çocuk</string>
   </property>
  </action>
  <action name="cikis">
   <property name="text">
    <string>Çıkış</string>
   </property>
  </action>
  <action name="Gris">
   <property name="text">
    <string>Griş Yap</string>
   </property>
  </action>
  <action name="kayit">
   <property name="text">
    <string>Kayıt </string>
   </property>
  </action>
  <action name="actionsepet">
   <property name="text">
    <string>sepet</string>
   </property>
  </action>
  <action name="actionFavoriler">
   <property name="text">
    <string>Favoriler</string>
   </property>
  </action>
  <action name="sepet_2">
   <property name="text">
    <string>sepet</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
